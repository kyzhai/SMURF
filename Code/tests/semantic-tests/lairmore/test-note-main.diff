5c5
< 	ID: print x :: a -> a
---
> 	ID: print  :: Unknown
