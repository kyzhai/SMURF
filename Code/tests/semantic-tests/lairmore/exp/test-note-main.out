Program: 
	SMain:  (1, 1)$2
Global Scope: 
	ID: main  :: Unknown
	ID: print  :: Unknown
	ID: random  :: Int

